docs/operations.rst

```diff
@@ -0,0 +1,10 @@
+Camus Operations
```

Comment:
Can you add a label here so I can link to this properly in the docs repo? Insert this on the first line:

```
.. _camus_operations:
```